internal/worker/server.go

```diff
@@ -410,6 +410,11 @@ func (s *Server) OSBuildKojiJob(id uuid.UUID, job *OSBuildKojiJob) error {
 	return nil
 }
 
+func (s *Server) JobChannel(id uuid.UUID) (string, error) {
```

Comment:
Just a thought: I wonder how many DB calls are we doing during handling all routes. We should probably start caching/rework our internal structures to lower the amount of queries. Definitely something for the future. Also, let's not optimize prematurely.

Reply:
I fully agree. I was thinking about the same thing and thought at least about not adding `JobChannel()`, but modifying `JobType()` to something like `JobInfo()` and returning more data about the job at once. But given that I ended up wrapping handlers in a middleware method which checks only the channel, it would not save us many calls.